test/index.test.js

```diff
@@ -386,4 +386,21 @@ describe('Fetch Tests', () => {
     const ts1 = Date.now();
     assert((ts1 - ts0) < 2000);
   });
+
+  it('fetch supports querystrings', async () => {
+    const EXPECTED = 'https://httpbin.org/delay/2?helix=dummy&foo=bar&rumple=stiltskin';
+    let res;
+    try {
+      const qs = {
+        helix: 'dummy',
+        foo: 'bar',
+        rumple: 'stiltskin',
+      };
+      res = await fetch('https://httpbin.org/delay/2', { qs });
+    } catch (err) {
+      assert.fail('Querystring functionality failed', err);
```

Comment:
why this catch? of the above throws, the test will fail anyways....

Reply:
@tripodsan I believe the code doesn't throw; and always returns

Reply:
so i should definitely fix this lol